lib/java_buildpack/framework/snyk.rb

```diff
@@ -0,0 +1,253 @@
+# frozen_string_literal: true
```

Comment:
Once rebased, run `bundle exec rake`.  You'll find that there are a handful of places where `rubocop` fails.  Please fix these.